Automattic/woocommerce-payments
# Disable Stripe Payment Element loader animation

#### Changes proposed in this Pull Request
This PR disables the UI animation from the Stripe Element loading skeleton. This UI animation was disabled in legacy UPE and split UPE. 

Before split UPE/deferred UPE projects, there was [this discussion](https://wcpay.wordpress.com/2022/05/13/update-to-the-payment-element-ui/) when the animation was first introduced. The conclusion was to disable this animation. 

Deferred UPE re-introduced it because we weren't aware of such a requirement.

|  Before this PR           | After |
--------------- | --------------- | 
|![Alt Text](https://cdn-std.droplr.net/files/acc_1260435/vm3BOp) | ![Alt Text](https://cdn-std.droplr.net/files/acc_1260435/f82HrH)









#### Testing instructions
Confirm that payment element loads without Stripe loader animation on the following pages:




* Regular checkout
* Blocks checkout
* My Account -> Add new payment method
* Pay for order 
* Subscription payment method change

-------------------


**Post merge**
